PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/ControlScript/MasterControl/TouchJump.rbxmx

```diff
@@ -55,8 +56,8 @@ function TouchJump:Create(parentFrame)
 	JumpButton.ImageRectOffset = Vector2.new(176, 222)
 	JumpButton.ImageRectSize = Vector2.new(174, 174)
 	JumpButton.Size = UDim2.new(0, jumpButtonSize, 0, jumpButtonSize)
-	JumpButton.Position = isSmallScreen and UDim2.new(1, jumpButtonSize * -2.25, 1, -jumpButtonSize - 20) or
-		UDim2.new(1, jumpButtonSize * -2.75, 1, -jumpButtonSize - 120)
+	JumpButton.Position = isSmallScreen and UDim2.new(1, -origJumpButtonSize - 10, 1, -origJumpButtonSize) or
```

Comment:
It was positioned a little high on an iPad, so I adjusted it to be level with the thumbstick. I tried it out on a few more devices and it looks good on all.